spark/src/test/scala/org/apache/spark/sql/CometTPCHQuerySuite.scala

```diff
@@ -76,7 +76,7 @@ class CometTPCHQuerySuite extends QueryTest with CometTPCBase with SQLQueryTestH
     "q20",
     "q21",
     "q22")
-  val disabledTpchQueries: Seq[String] = Seq("q1", "q15", "q17", "q18")
+  val disabledTpchQueries: Seq[String] = Seq()
```

Comment:
I re-ran these queries locally and they are all passed now. Let's see if we can have them in CI.